plugins/azure-devops/src/utils/isAzureLocation.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2023 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Entity } from '@backstage/catalog-model';
+import { AZURE_DEVOPS_PROJECT_LOCATION } from '../constants';
+
+export const isDevAzureLocation = (entity: Entity): boolean => {
```

Comment:
If we end up making this method https://github.com/backstage/backstage/blob/master/plugins/github-actions/src/components/getHostnameFromEntity.ts a core helper method then this won't be needed, however, if not we should mirror the functionality there in this file.